book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] [Optional] Please add Getters and Setters for @rentals. ie:
``attr_accessor :title, :author, :rentals``